cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -443,7 +448,20 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
         '''
         command: ceph auth get-or-create client.csi-cephfs-provisioner mon 'allow r' mgr 'allow rw' osd 'allow rw tag cephfs metadata=*'
         '''
-        cmd_json = {"prefix": "auth get-or-create",
+        metadata_pool = self._arg_parser.cephfs_metadata_pool_name
+        cluster_name = self._arg_parser.cluster_name
+        cmd_json = {}
+        if self._arg_parser.restricted_auth_permission:
+            if metadata_pool == "" or cluster_name == "":
```

Comment:
Actually I don't think cluster_name should be mandatory, let's just append it if it's there.

Reply:
I see the cluster name will be always there by default if not set , so just appended.
`common_group.add_argument("--cluster-name", default="openshift-storage",
                                  help="Ceph cluster name")`